src/components/ProjectSettingsTabs/PatchAliasesTab/getFormSchema.ts

```diff
@@ -27,6 +32,118 @@ export const getFormSchema = (
           patchAliasArray.schema
         ),
       },
+      patchTriggerAliases: {
+        title: "Patch Trigger Aliases",
+        ...overrideRadioBox(
+          "aliases",
+          [
+            "Override Repo Patch Trigger Aliases",
+            "Default to Repo Patch Trigger Aliases",
+          ],
+          {
+            type: "array" as "array",
+            items: {
+              type: "object" as "object",
+              properties: {
+                alias: {
+                  type: "string" as "string",
+                  title: "Alias",
```

Comment:
This doesn't really seem like it matters that much if it's hard to fix but I noticed that if I press tab from the Alias text box, I would expect it to take me to the next box (Project) but instead it takes me down to the Patch Alias toggle.

Reply:
Okay this is definitely a LeafyGreen bug! I've flagged it with them so hopefully it will be resolved at some point.